elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -35,26 +35,60 @@ export interface P2PPenalty {
 	readonly penalty: number;
 }
 
-export interface P2PPeerInfo {
+export interface P2PSharedState {
```

Comment:
I would prefer to not to have this interface and keep the values at the root. Please, assess how long will take to change the approach.